src/Pages/Login/index.scss

```
@@ -0,0 +1,252 @@
+
+        .bannerWrap {
+          margin: 37px 40px 0 0;
+
+          @media (max-width: 1250px) {
+            margin-right: 0;
+          }
+
+          @media (max-width: 1023px) {
+            display: none;
+          }
+
+          .info {
+            padding: 16px 9px 20px;
+
+            .strongTxt {
+              font-size: 25px;
+              line-height: 32px;
+              color: #252525;
+            }
+
+            .description {
+              margin-top: 5px;
+              color: #7c7c7c;
+              font-size: 13px;
+              font-weight: 600;
+              line-height: 18px;
+            }
+          }
+
+          img {
+            width: 540px;
+            object-fit: cover;
+          }
+        }
+        .formWrap {
+          display: inline-block;
+          width: 100%;
+          max-width: 480px;
+          padding: 60px 70px;
+          font-size: 16px;
+          text-align: left;
+          border: 1px solid #e5e5e5;
+
+          @media (max-width: 1250px) {
+            max-width: 420px;
+          }
+
+          .logo {
+            display: inline-block;
+            line-height: 1;
+            width: 120px;
+            height: 35px;
+            margin-bottom: 30px;
+            font-size: 30px;
+
+            p {
+              position: relative;
+
+              height: 100%;
+              z-index: -1;
+              overflow: hidden;
+            }
+          }
+
+          .formContainer {
+            input {
+              width: 100%;
+              height: 48px;
+              border: 2px solid;
+              border-style: solid;
+              border-width: 0 0 2px 0;
+              border-color: #ebebeb;
+              padding: 11px 0 8px 0;
+              color: #252525;
+              font-size: 15px;
+              font-weight: 600;
+              outline: 0;
+              caret-color: #f7e317;
+
+              &::placeholder {
+                color: #8e8e8e;
+                letter-spacing: -1px;
+              }
+
+              &[type='password'] {
+                letter-spacing: 1px;
+              }
+            }
+
+            .keepLogin {
+              margin-top: 25px;
+
+              label {
+                font-size: 14px;
+                line-height: 20px;
+                font-weight: 600;
+                color: #4c4c4c;
+                cursor: pointer;
+
+                .keepLoginInBox {
+                  width: 20px;
+                  height: 20px;
+                  margin: 0 8px 0 0;
+                  border-radius: 50%;
+                  vertical-align: text-bottom;
+
+                  &[type='checkbox']:checked {
+                    background-color: #d9534f;
+                    color: #a4b7c6;
+                    border-radius: 50%;
+                  }
+                }
+              }
+            }
+
+            .loginLink {
+              .loginBtn {
+                display: block;
+                width: 100%;
```

```suggestion
              .loginBtn {
                display: block;
                width: 100%;
```
스타일이 없는 셀렉터는 굳이 표기해줄 필요 없을거 같아요

---

수정했습니다.